### DISCUSSION TITLE
Bitdevs en Español #003

### DISCUSSION THREAD

**Discussion by lucasdcf** [bitdevsespanol/bitdevsespanol.org#2] [General]
Hola chicos, creé esta discusión aquí para que envíen los temas que desean discutir en los próximos Bitdevs. Que seyan cosas que se discutieron en los ultimos 2/3 meses en la comunidad de desarrollo de Bitcoin.
Envía también los enlaces. Y si hay algun tema que el que te sientes muy cómodo, puedes ser voluntario para presentar también :)
Si te gusta el tema del colega, puene un 👍

**Comment by dansecret** [comment]
.
